ios/Classes/FlutterInappPurchasePlugin.m

```diff
@@ -365,8 +365,6 @@ -(NSDictionary *)getProductObject:(SKProduct*)product{
                   break;
           }
 
-          introductoryPriceNumberOfPeriods = [@(product.introductoryPrice.numberOfPeriods) stringValue];
```

Comment:
Is removing fixing the value of the variable?

Reply:
You set the value of `introductoryPriceNumberOfPeriods` just before this line in the switch.

Reply:
Oh! Thanks for clarifying this. I'll release to newer version!